tests/python/unittest/test_random.py

```diff
@@ -18,6 +18,7 @@
 import os
 import mxnet as mx
 import numpy as np
+from nose.plugins.attrib import attr
```

Comment:
unused import

Reply:
ty

Reply:
Actually attr is used on line 192 right?

Reply:
oops, wrong file. I meant for the sparse tests.

Reply:
Gotcha.   Updated.